src/Service/ActivityMonitor.php

```diff
@@ -198,7 +198,10 @@ private function parseLog($stream, &$buffer) {
      */
     private function canRead($stream, $microseconds) {
         $readSet = [$stream];
-        $ignore = [];
+
+        // PHP expects NULL, not an empty array, to represent no streams.
```

Comment:
This looks wrong, the linked documentation is clear: PHP expects an empty array.

But parameters should be different variables (in practice it doesn't really matter but this is a wrong statement, and idk how PHP reacts to that).

Reply:
The error is not on this line, the error is that the stream is invalid. It's actually said in the error: `Cannot cast a filtered stream on this system`.

Reply:
That's an error that I don't understand... but also, it was just a warning.

The fatal error was `Uncaught ValueError: No stream arrays were passed`